templates/mail.publication.text.php

```diff
@@ -1,13 +1,11 @@
 <?php p($l->t('Hello,')); ?>
-<?php p('\n\n'); ?>
 
-<?php $l->t("We wanted to inform you that %s has published the calendar %s.", [$_['username'],$_['calendarname']]); ?>
-<?php p('\n'); ?>
 
-<?php p($l->t('Click on the link below to access it !')); ?>
-<?php p('\n'); ?>
```

Comment:
@tcitworld Just as a side note:
When using `\n` in PHP you have to you doubles quotes. `'\n'` won't work.
And as this is a template, you can just insert actual new lines :)